### DISCUSSION TITLE
Allow users to choose which browser's bookmarks to load

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#515] [Ideas]
**Is your feature request related to a problem? Please describe.**

Allow users to choose which browser's bookmarks to load. Currently supported browsers are Firefox, Chromium Edge, Internet Explorer and Chrome.

This issue is transferred from https://github.com/jjw24/Wox/issues/97

![image](https://user-images.githubusercontent.com/26427004/87896515-c9047980-ca8b-11ea-89dc-1584fd5442fc.png)

First request has been completed.

This issue is for the second request:
Choose which browser to search bookmarks 

**Describe the solution you'd like**

An option in the form of check boxes to allow user to select one, two etc or all browsers' bookmarks. Current default is all browser's bookmarks.

**Comment by jjw24** [comment]
@taooceros this can probably also move back to issues as there is not much discussion to it right?

**Reply by taooceros** [reply]
yeah you are right, though I think we may tackle it after 1.8.0.
